pkg/ttnpb/gatewaytokens.pb.paths.fm.go

```diff
@@ -0,0 +1,35 @@
+// Code generated by protoc-gen-fieldmask. DO NOT EDIT.
```

Comment:
The general code quality CI failure is because of this file not having our copyright header. It's an issue with these protoc plugins that they don't copy the proto's copyright header to the output file, but let's not bother.

Reply:
Can we skip these in the mage commands?